### PR TITLE
Remove unnecessary documentation of non-existent constructors

### PR DESCRIPTION
**Pull request by kocsismate**
Following comment https://github.com/php/php-src/pull/10127#issuecomment-1365101207

### CODE REVIEW

File: reference/spl/splqueue/construct.xml [hunk 1]
```diff
@@ -1,105 +0,0 @@
...
-   </note>
-  </para>
- </refsect1>
-
- <refsect1 role="parameters">
-  &reftitle.parameters;
-  &no.function.parameters;
- </refsect1>
-
- <refsect1 role="examples">
```
Comment (cmb69) [inline reference/spl/splqueue/construct.xml:31]: Should move the examples to the `SplQueue` class page?

File: reference/spl/splstack/construct.xml [hunk 2]
```diff
@@ -1,83 +0,0 @@
...
-   </note>
-  </para>
- </refsect1>
-
- <refsect1 role="parameters">
-  &reftitle.parameters;
-  &no.function.parameters;
- </refsect1>
-
- <refsect1 role="examples">
```
Comment (cmb69) [inline reference/spl/splstack/construct.xml:31]: We should probably move this example to the `SplStack` class page (and adapt the title of the example).

**Review by cmb69 — COMMENTED** [review]
Thank you! Besides that we probably want to keep the examples, this is fine.

File: reference/spl/splstack/construct.xml (hunk 2 above)
Comment (kocsismate) [reply reference/spl/splstack/construct.xml:31]: done!

File: reference/spl/splqueue/construct.xml (hunk 1 above)
Comment (kocsismate) [reply reference/spl/splqueue/construct.xml:31]: done!

File: reference/spl/splstack.xml [hunk 3]
```diff
@@ -69,6 +67,36 @@
    </variablelist>
   </section>
 }}} -->
+
+  <refsect1 role="examples">
```
Comment (cmb69) [inline reference/spl/splstack.xml:71]:
```suggestion
  <section role="examples">
```

File: reference/spl/splqueue.xml [hunk 4]
```diff
@@ -70,6 +68,55 @@
...
   </section>
 }}} -->
+
+  <refsect1 role="examples">
+   &reftitle.examples;
+   <para>
+    <example>
+     <title><classname>SplQueue</classname> example</title>
+     <programlisting role="php">
+      <![CDATA[
```
Comment (cmb69) [inline reference/spl/splqueue.xml:78]: We do not indent `<![CDATA[` brackets; thus:
```suggestion
<![CDATA[
```

Comment (cmb69) [inline reference/spl/splqueue.xml:78]: We do not indent `<![CDATA[` brackets; thus:
```suggestion
<![CDATA[
```

File: reference/spl/splqueue.xml [hunk 5]
```diff
@@ -70,6 +68,55 @@
...
+$q[] = 3;
+foreach ($q as $elem)  {
+ echo $elem."\n";
+}
+?>
+]]>
+     </programlisting>
+     &example.outputs;
+     <screen>
+      <![CDATA[
```
Comment (cmb69) [inline reference/spl/splqueue.xml:92]:
```suggestion
<![CDATA[
```

File: reference/spl/splqueue.xml [hunk 6]
```diff
@@ -70,6 +68,55 @@
...
+1
+2
+3
+]]>
+     </screen>
+    </example>
+    <example>
+     <title>Efficiently handling tasks with <classname>SplQueue</classname></title>
+     <programlisting role="php">
+      <![CDATA[
```
Comment (cmb69) [inline reference/spl/splqueue.xml:102]:
```suggestion
<![CDATA[
```

File: reference/spl/splstack.xml [hunk 7]
```diff
@@ -69,6 +67,36 @@
...
+   &reftitle.examples;
+   <para>
+    <example>
+     <title><classname>SplStack</classname> example</title>
+     <programlisting role="php">
+      <![CDATA[
```
Comment (cmb69) [inline reference/spl/splstack.xml:77]:
```suggestion
<![CDATA[
```

File: reference/spl/splstack.xml [hunk 8]
```diff
@@ -69,6 +67,36 @@
...
+$q[] = 3;
+foreach ($q as $elem)  {
+ echo $elem."\n";
+}
+?>
+]]>
+     </programlisting>
+     &example.outputs;
+     <screen>
+      <![CDATA[
```
Comment (cmb69) [inline reference/spl/splstack.xml:91]:
```suggestion
<![CDATA[
```

File: reference/spl/splqueue.xml [hunk 9]
```diff
@@ -70,6 +68,55 @@
    </variablelist>
   </section>
 }}} -->
+
+  <refsect1 role="examples">
```
[the remark this answers is not in the document]
Comment (kocsismate) [reply reference/spl/splqueue.xml]: Ah, I see, thanks! I tried to fix the issue the other way around (move out the section from partintro)

[the remark this answers is not in the document]
Comment (kocsismate) [reply reference/spl/splqueue.xml]: Fixed now!

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**